common/client-core/src/client/real_messages_control/real_traffic_stream.rs

```diff
@@ -259,6 +265,13 @@ where
 
         if let Err(err) = self.mix_tx.send(vec![next_message]).await {
             log::error!("Failed to send: {err}");
+        } else {
+            let event = if fragment_id.is_some() {
+                PacketStatisticsEvent::RealPacketSent
+            } else {
+                PacketStatisticsEvent::CoverPacketSent
```

Comment:
I wonder if it could be useful to separate cover messages generated from the cover traffic stream vs real traffic stream

Reply:
Well, I thought about it, but since we can't separate them when receiving I thought I'd keep it symmetric